pkg/metrics/metrics.go

```diff
@@ -10,10 +10,17 @@ var (
 		Name:      "reconfigured_total",
 		Help:      "Total times the image registry's storage was reconfigured.",
 	})
+	imagePrunerInfo = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "image_registry_operator",
+		Subsystem: "image_pruner",
+		Name:      "install_info",
+		Help:      "Installation status code related to the automatic image pruning feature. 0 = not installed, 1 = suspended, 2 = enabled",
+	}, []string{})
```

Comment:
There is no need for this to be a GaugeVec if there are no labels, and the `WithLabelValues` call is also not needed in that case.